pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "libertem-icom"
+description = "Integrated center of mass / differential phase contrast following https://doi.org/10.1038/s41598-018-20377-2"
+license = {file = "LICENSE"}
+keywords = ["phase contrast", "LiberTEM", "center of mass", "differential phase contrast"]
+requires-python = ">=3.7"
+dynamic = ["version", "readme"]
+dependencies = [
+    "numpy",
+    "libertem>0.11",
```

Comment:
As this depends on the CoM UDF, should the LiberTEM dependency be `>=0.12.dev0`?

Reply:
Yes, of course! :-) Thx